internal/output/table.go

```diff
@@ -84,6 +90,55 @@ func tableBuilder(outputTable table.Writer, vulnResult *models.VulnerabilityResu
 	return outputTable
 }
 
+func printContainerScanningResult(result Result, outputWriter io.Writer, terminalWidth int) {
+	summary := fmt.Sprintf(
+		"Total %[1]d packages affected by %[2]d vulnerabilities (%[3]d Critical, %[4]d High, %[5]d Medium, %[6]d Low, %[7]d Unknown) from %[8]d ecosystems, %[9]d have fixes available",
```

Comment:
nit: "%d have fixes available" -- are these packages or vulnerabilities? We should clarify.

Probably also better to start it on its own sentence. i.e. ". X <packages?/vulnerabilities?> have fixes available."